api/src/main/java/io/opentelemetry/metrics/Metric.java

```diff
@@ -24,41 +24,41 @@
 /**
  * Base interface for all metrics defined in this package.
  *
- * @param <H> the Handle.
+ * @param <B> the Bound.
  * @since 0.1.0
  */
 @ThreadSafe
-public interface Metric<H> {
+public interface Metric<B> {
   /**
-   * Returns a {@code Handle} with associated with specified {@code labelSet}. Multiples requests
-   * with the same {@code labelSet} may return the same {@code Handle}.
+   * Returns a {@code Bound} with associated with specified {@code labelSet}. Multiples requests
+   * with the same {@code labelSet} may return the same {@code Bound}.
    *
-   * <p>It is recommended to keep a reference to the Handle instead of always calling this method
-   * for every operations.
+   * <p>It is recommended to keep a reference to the Bound instead of always calling this method for
+   * every operations.
    *
    * @param labelSet the set of labels.
-   * @return a {@code Handle} the value of single gauge.
+   * @return a {@code Bound} the value of single gauge.
    * @throws NullPointerException if {@code labelValues} is null.
    * @since 0.1.0
    */
-  H getHandle(LabelSet labelSet);
+  B getBound(LabelSet labelSet);
```

Comment:
this method should be `bind` per the OTEP: https://github.com/open-telemetry/oteps/blob/master/text/0070-metric-bound-instrument.md#internal-details

Reply:
Will do in a separate PR. Need to merge this because it got large.